packages/stores/types.ts

```diff
@@ -234,11 +247,36 @@ type TUiStore = {
     toggleLanguageSettingsModal: () => void;
     toggleReadyToDepositModal: () => void;
     toggleSetCurrencyModal: () => void;
+    removeToast: () => void;
```

Comment:
@mahdiyeh-deriv Same here, Please update the types correctly as other devs gonna rely on it later on 🙏🏻